### ISSUE TITLE
When tabbing and you move to a new column the user is not informed of the new column

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vsts-extension-retrospectives#1073]
### Tags:
#A11y;  #accessibility;

### Repro Steps:
Navigate to [Retrospectives - Boards](https://dev.azure.com/eDADPlayground/SCRUMProject/_apps/hub/ms-devlabs.team-retrospectives.home#teamId=1994b7b8-6d1b-4c29-9d29-46c65d6657cd&boardId=84220a19-a3c7-450e-986c-a242dd5e787e).
Navigate with Screen reader on the columns.

### Actual Result:
When tabbing and you move to a new column the user is not informed of the new column.

### Expected Result:
When user moves to next column then the column name should be announced.

### Attachment:

![Image](https://github.com/user-attachments/assets/4d07196f-cc68-4a0e-aef4-1936f6d2657c)

**Comment by msftedad** [comment]
#A11y #accessibility

**Comment by polatengin** [comment]
This has been fixed in #1386. Screen readers now announce the column name and feedback count when focus moves to a new column. I am closing this issue as completed.
